lib/curl_tests.py

```diff
@@ -229,18 +233,19 @@ def test(
             elif asset.startswith(f"{domain}/"):
                 asset = asset.replace(f"{domain}/", "")
             if not asset.startswith("/"):
-                asset = "/" + asset
+                asset = urljoin(base + "/", asset)
+            resolved_asset_url = f"https://{domain}{asset}"
```

Comment:
What may work is using `urljoin` specifically here:
```suggestion
            resolved_asset_url = urljoin("https://{domain}", asset)
```

Wdyt?

Reply:
To be honest, I don't remember why I committed that. If I am not wrong, `domain` should only contain a domain as its name suggest. So this should be safe.

Reply:
Indeed it should be safe. Lezdoit.

Reply:
I checked in run against my package, [Grist](https://github.com/YunoHost-Apps/grist_ynh), it seems to work (it passed the Installation steps, but not the upgrades or backup/restore, probably due to instabilities when running locally).